### PR TITLE
Additional Rexel spiders

### PR DESCRIPTION
**Pull request by peternewman**
After #4708 these will mostly need QIDs

### CODE REVIEW

File: locations/spiders/ideal_au.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from locations.storefinders.rexel import RexelSpider
+
+
+class IdealAUSpider(RexelSpider):
+    name = "ideal_au"
+    item_attributes = {"brand": "Ideal Electrical (Australia)", "brand_wikidata": "Q"}
```
Comment (Cj-Malone) [inline locations/spiders/ideal_au.py:6]:
```suggestion
    item_attributes = {"brand": "Ideal", "brand_wikidata": "Q116694781"}
```

File: locations/spiders/ideal_nz.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from locations.storefinders.rexel import RexelSpider
+
+
+class IdealNZSpider(RexelSpider):
+    name = "ideal_nz"
+    item_attributes = {"brand": "Ideal (New Zealand)", "brand_wikidata": "Q"}
```
Comment (Cj-Malone) [inline locations/spiders/ideal_nz.py:6]:
```suggestion
    item_attributes = {"brand": "Ideal", "brand_wikidata": "Q116694781"}
```

File: locations/spiders/westburne_ca.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from locations.storefinders.rexel import RexelSpider
+
+
+class WestburneCASpider(RexelSpider):
+    name = "westburne_ca"
+    item_attributes = {"brand": "Westburne (Canada)", "brand_wikidata": "Q"}
```
Comment (Cj-Malone) [inline locations/spiders/westburne_ca.py:6]:
```suggestion
    item_attributes = {"brand": "Westburne", "brand_wikidata": "Q116694882"}
```

File: locations/spiders/rexel_de.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+from locations.storefinders.rexel import RexelSpider
+
+
+class RexelDESpider(RexelSpider):
+    name = "rexel_de"
+    item_attributes = {"brand": "Rexel (Germany)", "brand_wikidata": "Q962489"}
```
Comment (Cj-Malone) [inline locations/spiders/rexel_de.py:6]:
```suggestion
    item_attributes = {"brand": "Rexel", "brand_wikidata": "Q962489"}
```

**Comment by davidhicks** [comment]
I had a look at improving the Rexel spiders too (and adding a bunch of missing countries/brands).

**Some observations**
- For Australian brands, you get Rexel, Ideal, Turks, Lear & Smith's store locations all returned from the same store finder being https://www.rexel.com.au/are/store-finder/findNearbyStores
- For Australia, multiple coordinates need to be supplied as results returned appear to be limited to those with a distance less than ~170-180km. 
- A @data-stores attribute on the store-finder page contains all stores with coordinates, but I found, depending on brand/country, it may be missing a few minor details versus the findNearbyStores API e.g. lack of e-mail addresses for each store.

**Proposal**
I think we could try an alternative approach for the Rexel storefinder:
1. Scrape the store-finder page for the @data-stores attribute.
2. Create an array of coordinates of all stores.
3. Loop until the array of store coordinates has no elements left:
3a. Call findNearbyStores and yield X results (X changes per country/brand).
3b. Check if findNearbyStores results are paginated and if so, yield the second and subsequent pages of results.
3c. Remove coordinates of returned stores from the array of coordinates.

This proposed spider could pick coordinates in 3 heavily populated areas, yielding 150 out of the 200 stores total that may exist. Then the remaining 50 stores in the array of coordinates could potentially be worked through with an additional 20 queries to the findNearbyStores API call.

**Pull request closed without merging**